src/Services/Air/AirFormat.js

```diff
@@ -95,6 +106,8 @@ function formatSegment(segment) {
     airline: segment.Carrier,
     flightNumber: segment.FlightNumber,
     uapi_segment_ref: segment.Key,
+    isCodeshare,
+    ...codeshareInfo
```

Comment:
I would either to keep codeshareInfo as structure here or to prepend codeshare keys with `codeshare`.
First option sounds more preferable to me.

Reply:
Used Object.assign to add codeshare informations when isCodeshare is true